.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 sudo: false
 node_js:
-  - "0.10"
+  - "6.2.10"
```

Comment:
@rclark Ah good point, this is habit as some of our C deps in geocodeladia have to be fixed on this. Not necessary here - anything `6.x` should be fine as long as osmium installs cleanly.